internal/devices/update.go

```diff
@@ -37,14 +38,14 @@ func (c *Client) Update() *cobra.Command {
 		userdata      string
 		hostname      string
 		tags          []string
-		alwaysPXE     bool
```

Comment:
We should be able to use a `bool` here. To check if the variable was explicitly set, we should be able to use `cmd.Flags("always-pxe").Changed()`

https://pkg.go.dev/github.com/spf13/pflag#FlagSet.Changed